### PR TITLE
Add migration extensions to docs

### LINKED ISSUE #634: Keeping Django database migrations backward compatible
After reading this post: https://medium.com/3yourmind/keeping-django-database-migrations-backward-compatible-727820260dbb

Maybe we should add next libraries:
* https://github.com/3YOURMIND/django-add-default-value
* https://github.com/3YOURMIND/django-deprecate-fields
* https://github.com/3YOURMIND/django-migration-linter

### PR DESCRIPTION
**Pull request by DmytroLitvinov**
Closes #634

### CODE REVIEW

File: {{cookiecutter.project_name}}/docs/pages/template/django.rst [hunk 1]
```diff
@@ -120,6 +120,9 @@ Here's a full list of the extensions for both development and production:
 - `django-referrer-policy`_ - middleware implementing the `Referrer-Policy`_
 - `django-health-check`_ - checks for various conditions and provides reports
   when anomalous behavior is detected
+- `django-add-default-value`_ - this django Migration Operation can be used to transfer a Fields default value to the database scheme.
```
Comment (sobolevn) [inline {{cookiecutter.project_name}}/docs/pages/template/django.rst]: We don't use `.` at the end of the list item

**Comment by DmytroLitvinov** [comment]
Refactored.

File: {{cookiecutter.project_name}}/docs/pages/template/django.rst [hunk 2]
```diff
@@ -120,7 +120,7 @@ Here's a full list of the extensions for both development and production:
 - `django-referrer-policy`_ - middleware implementing the `Referrer-Policy`_
 - `django-health-check`_ - checks for various conditions and provides reports
   when anomalous behavior is detected
-- `django-add-default-value`_ - this django Migration Operation can be used to transfer a Fields default value to the database scheme.
+- `django-add-default-value`_ - this django Migration Operation can be used to transfer a Fields default value to the database scheme
 - `django-deprecate-fields`_ - this package allows deprecating model fields and allows removing them in a backwards compatible manner.
```
Comment (sobolevn) [inline {{cookiecutter.project_name}}/docs/pages/template/django.rst]: Extra `.` here

File: {{cookiecutter.project_name}}/docs/pages/template/django.rst [hunk 3]
```diff
@@ -120,7 +120,7 @@ Here's a full list of the extensions for both development and production:
...
 - `django-migration-linter`_ - detect backward incompatible migrations for your django project.
```
Comment (sobolevn) [inline {{cookiecutter.project_name}}/docs/pages/template/django.rst]: And here 🙂

File: {{cookiecutter.project_name}}/docs/pages/template/django.rst (hunk 2 above)
Comment (DmytroLitvinov) [reply {{cookiecutter.project_name}}/docs/pages/template/django.rst]: It is outdated ;)

File: {{cookiecutter.project_name}}/docs/pages/template/django.rst (hunk 3 above)
Comment (DmytroLitvinov) [reply {{cookiecutter.project_name}}/docs/pages/template/django.rst]: And this one is outdated.

**Comment by sobolevn** [comment]
@DmytroLitvinov any progress? 🙂

**Comment by sobolevn** [comment]
Thanks a lot!

**Pull request merged**